src/Evolution/Systems/RadiationTransport/M1Grey/M1Closure.cpp

```diff
@@ -0,0 +1,270 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Evolution/Systems/RadiationTransport/M1Grey/M1Closure.hpp"
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Variables.hpp"
+#include "NumericalAlgorithms/RootFinding/NewtonRaphson.hpp"
+#include "PointwiseFunctions/GeneralRelativity/IndexManipulation.hpp"
+#include "Utilities/ConstantExpressions.hpp"
+#include "Utilities/Gsl.hpp"
+
+/// \cond
+namespace {
+
+struct LorentzFactorSquared {
+  using type = Scalar<DataVector>;
+};
+struct VelocitySquared {
+  using type = Scalar<DataVector>;
+};
+struct VelocityLow {
+  using type = tnsr::i<DataVector, 3, Frame::Inertial>;
+};
+struct MomentumUp {
```

Comment:
Are these `{\tilde S}^m` and `{\tildeS}^2` from the Valencia system or something else?  If the former we should probably add these to ValenciaDivClean/Tags.hpp and reuse them in that directory (not something you need to fix here)

Reply:
Yes, I can move them if we agree on a name.

Reply:
Ah okay, then we can't reuse the tags.